website/content/en/preview/AWS/provisioning.md

```diff
@@ -358,3 +358,26 @@ Refer to general [Kubernetes GPU](https://kubernetes.io/docs/tasks/manage-gpus/s
 * `amd.com/gpu`: [AMD GPU device plugin for Kubernetes](https://github.com/RadeonOpenCompute/k8s-device-plugin)
 * `aws.amazon.com/neuron`: [Kubernetes environment setup for Neuron](https://github.com/aws/aws-neuron-sdk/blob/master/neuron-deploy/tutorials/tutorial-k8s.rst)
 {{% /alert %}}
+
+### Pod ENI (Security Groups for Pods)
+[Pod ENI](https://github.com/aws/amazon-vpc-cni-k8s#enable_pod_eni-v170) is a feature of the AWS VPC CNI Plugin which allows an Elastic Network Interface (ENI) to be allocated directly to a Pod. When enabled, the `vpc.amazonaws.com/pod-eni` extended resource is added to supported nodes. Pod ENI support must be enabled to use the Security Groups for Pods feature. To allow Karpenter to provision nodes with the `vpc.amazonaws.com/pod-eni` resource, you must enable Pod ENI support in both the AWS VPC CNI plugin as well as in Karpenter.
```

Comment:
> Pod ENI support must be enabled to use the Security Groups for Pods feature

Link here to show the user how to enable it?

Reply:
There are two sections below which describe how to enable pod-eni in the VPC CNI plugin as well as in Karpenter. I also linked to the Security Groups for Pods docs site at the bottom.  Perhaps I could move it up to this line instead.

Reply:
Ah, got it. Yeah, probably no need for links then.

Still, since the below section is describing the process of how to enable this in your cluster, we should phrase these as steps. i.e.

### Enable Security Groups for Pods
1. Enable Pod ENI support in the AWS VPC CNI plugin
2. Enable Pod ENI support in Karpenter

Reply:
Updated now.  Take a look.